compiler/rustc_ast/src/util/parser.rs

```diff
@@ -287,6 +287,7 @@ pub enum ExprPrecedence {
     Struct,
     Gen,
     Await,
+    Unreachable,
```

Comment:
Can you just give `ExprKind::Unreachable` a dummy precedence? Like, just give it the same precedence as `Lit` or something.